assets/dev/js/editor/controls/behaviors/global-select-behavior.js

```diff
@@ -0,0 +1,136 @@
+export default class GlobalControlSelect extends Marionette.Behavior {
+	ui() {
```

Comment:
getCurrentStatus: () = {
	return !! this.view.container.globals.get( this.view.model.get( 'name' ) );
}